ProcessQueue/__init__.py

```diff
@@ -266,6 +267,294 @@ def process_failure(api_url, scan_report_id, headers):
         headers=headers
     )
 
+def paginate_chars(entries_to_post,other):
```

Comment:
I think the cleanest solution is to remove this new function, and then modify the `paginate` function. Add a second argument `other=''`, and then it can be called by either `paginate(entries)` or `paginate(entries, other)`, and in the former case `len('')` will evaluate to `0`.

Reply:
Good point. Could you do it this way please:
```
def paginate(entries_to_post, max_chars=None):
    """
    This expects a list of dicts, and returns a list of lists of dicts, 
    where the maximum length of each list of dicts, under JSONification, 
    is less than max_chars
    """
    if not max_chars:
        max_chars = int(os.environ.get("PAGE_MAX_CHARS")) if os.environ.get("PAGE_MAX_CHARS") else 10000
```
Then in the POST case you don't need to make any change, but in the GET case you just supply `max_chars=2000` as an extra argument. Make a note of that in the docstring too please.

Reply:
done